daemon/src/engine/qcommon/q_math.cpp

```diff
@@ -1548,6 +1392,7 @@ void ProjectPointOntoVectorBounded( vec3_t point, vec3_t vStart, vec3_t vEnd, ve
 	}
 }
 
+// Unused, equivalent implementation found in cm_trace.cpp as CM_DistanceFromLineSquared
```

Comment:
Looking more closely, the helper functions in CM have a different implementations than in non CM despite sharing the same name and arguments...